### PR TITLE
add bookstore app UI

### PR DESCRIPTION
**Pull request by Bifabig**
- **Use Functional Components**
- **Implement Routing using React Router**
- **Style using CSS modules**

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { Routes, Route } from 'react-router-dom';
+import Books from './Books';
+import Navbar from './Navbar';
+import '../styles/App.module.scss';
+
+const BookstoreApp = () => (
+  <Routes>
+    <Route path="/" element={<Navbar />}>
+      <Route index element={<Books />} />
```
Comment (Kamba56) [inline src/components/App.js:10]: - [x] Great job adding route to the books component. Kindly add a route to the categories element. Also, add the categories component as demanded by project requirements.
image

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @Bifabig, 

Good job so far!!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [x] Great job adding styles to your component!👏👏👏
- [x] Great use of react router

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/App.js (hunk 1 above)
Comment (Bifabig) [reply src/components/App.js:10]: Hi @Kamba56, Kindly review it again... I've made the necessary changes.

Comment (peterrobert) [reply src/components/App.js:10]: This change has been done well

**Review by peterrobert — APPROVED** [review]
Hi @Bifabig, 


### Highlights!

[ ] Good feature branch name ✔️
[ ] Descriptive pull request ✔️


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**